client/src/pages/SignIn/index.tsx

```diff
@@ -1,5 +1,103 @@
+import background from "../../assets/signin-bg-img.jpg";
+import logo from "../../assets/logoM.png";
+import {useForm} from 'react-hook-form'
+import {DevTool} from "@hookform/devtools"
+import { FacebookIcon, AppleIcon, GoogleIcon} from "../../assets/icons/icons";
+import { Link  } from "react-router-dom";
+import { Button } from "../../components/Button";
+
+interface SignInFormTypes {
+  username: string;
+  password: string;
+}
+
 const SignIn = () => {
-  return <div>SignIn</div>;
+  // const navigate = useNavigate();
+  const form = useForm<SignInFormTypes>()
+  const {register, handleSubmit, control, watch, formState :{errors}} =form
+
+  const username = watch("username")
+  const password = watch("password")
+  const onSubmit = (data:SignInFormTypes) =>{
+    username === data.username && password === data.password ? "": ""
+
+  }
+
+  return <div className="flex">
+    {/* bg-cover h-full w-full absolute bg-no-repeat z-auto */}
+    <img src={background} alt="banner" className=" bg-cover block overflow-hidden  h-full w-full absolute bg-no-repeat z-auto" />
+    <div className="hidden absolute sm:flex top-[3%] left-[2%] flex items-center mr-2">
+      <img alt="logo" src={logo} className=" p-2 mr-2 rounded-2xl bg-orange-primary"/>
+      <h3 className="text-base font-semibold sm:text-xl">homez</h3>
+    </div>
+    
+    <form className="w-full sm:w-[539px] h-full sm:h-auto rounded-lg flex-col absolute bg-white p-10 bg-opacity-[75%] right-0 sm:top-[10%] sm:mr-[5%]" 
+    onSubmit={handleSubmit(onSubmit)} noValidate >
+      {/* head */}
+        <div>
+          <div className="flex justify-between">
+            <div className="flex">
+              <p className="flex-initial text-[22px] font-medium">Welcome to </p>
+              <Link to="/home" className="flex-initial px-[3px] text-[22px] font-medium text-orange-primary">Homez</Link>
+            </div>
+            <div className="flex flex-col">
+              <span className="text-[14px]">No Account?</span>
+              <Link to="/signup" className="text-orange-primary text-[14px]">Sign Up</Link>
+            </div>
+          </div>
+        </div>
+        {/* Sign in others */}
+        <h1 className="font-semibold text-5xl">Sign in</h1>
+        <div className="flex py-14 justify-between">
+          <Button className="flex h-[55px] w-[298px] bg-[--google-button-background] font-medium rounded-lg text-blue-400 items-center justify-center">
```

Comment:
Vả lại cái button này, nếu dùng ở 2 nơi trở lên thì vô config cái **varient** vs **size** của nó mà gọi ra thôi, chứ không dùng như này
-**google-button-background** : Tên biến này không đặt như này, nó màu gì thì ghi ra. Ví dụ: blue-primary (secondary), etc...
```suggestion
          <Button className="flex h-14 w-72 bg-blue-secondary font-medium rounded-lg text-blue-400 items-center justify-center">
```